projects/tests/test_project.py

```diff
@@ -26,6 +26,18 @@ def setUp(self):
         )
         _ = Project.objects.create_project(name="test-perm", owner=user, description="")
         user = User.objects.create_user(test_member["username"], test_member["email"], test_member["password"])
+        self.flavor = Flavor.objects.create(
+            cpu_req="500m",
+            cpu_lim="1000m",
+            mem_req="0.5Gi",
+            mem_lim="1Gi",
+            ephmem_req="200Mi",
+            ephmem_lim="500Mi",
+            gpu_req="1",
+            gpu_lim="1",
+            name="1 vCPU, 0.5 GB RAM",
```

Comment:
```suggestion
            name="1 vCPU, 0.5 GB RAM, 1 GPU",
```

Which model by the way, I keep forgetting